ast/ddl.go

```diff
@@ -530,6 +530,48 @@ func (n *TableToTable) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+//CreateViewStmt is a statement to create a View.
```

Comment:
Leave a blank after //

Reply:
Please update other similar lines in `executor/ddl_test.go`. @WenBoYanging